joern-cli/frontends/pysrc2cpg/src/main/scala/io/joern/pysrc2cpg/PythonAstVisitorHelpers.scala

```diff
@@ -278,7 +278,8 @@ trait PythonAstVisitorHelpers { this: PythonAstVisitor =>
     name: String,
     lineAndColumn: LineAndColumn,
     argumentNodes: Iterable[NewNode],
-    keywordArguments: Iterable[(String, NewNode)]
+    keywordArguments: Iterable[(String, NewNode)],
+    methodFullName: String
```

Comment:
Instead of passing around blank strings, rather use `Option` types.
```suggestion
    methodFullName: Option[String] = None
```